src/host/getset.cpp

```diff
@@ -1341,7 +1341,6 @@ void DoSrvPrivateShowCursor(SCREEN_INFORMATION& screenInfo, const bool show) noe
 void DoSrvPrivateAllowCursorBlinking(SCREEN_INFORMATION& screenInfo, const bool fEnable)
 {
     screenInfo.GetActiveBuffer().GetTextBuffer().GetCursor().SetBlinkingAllowed(fEnable);
-    screenInfo.GetActiveBuffer().GetTextBuffer().GetCursor().SetIsOn(!fEnable);

```

Comment:
oh interesting, so the old code actually fully disabled the cursor if blinking was disabeld.

i want to make sure, though: _if the user turns blinking off while the cursor is visible, will it get stuck on? if it's off, will it get stuck off?_

Reply:
From what I've gathered from other terminals, when blinking is disabled, the cursor should remain `On` always, and have the visibility controlled by the `IsVisible` property. So when you do a `printf "\e[?12l"` to disable blinking, the cursor stays stuck `On`. At this point, only the cursor visibility property controls whether the user can see it or not. (Yes, the cursor can be `On` and NOT `Visible`)

Reply:
I don’t understand why it gets stuck _on_ though. What if it’s off (blinked-off) when the blink timer is arrested?

Reply:
That's a good question - it does seem to get stuck `On`, but if this isn't triggering it, then something else must be. I'll write a test to make sure. Because it _should_ be on.